apps/web/vite.config.ts

```diff
@@ -27,11 +27,14 @@ export default defineConfig(({ mode }) => {
       include: ['src/**/*.{test,spec}.{js,ts}']
     },
     optimizeDeps: {
-      exclude: ['@node-rs/argon2', '@node-rs/bcrypt']
+      exclude: ['@node-rs/argon2', '@node-rs/bcrypt', 'fsevents']
     },
     build: {
       commonjsOptions: {
         include: [/@tableslayer\/ui/, /node_modules/]
+      },
+      rollupOptions: {
+        external: ['fsevents']
```

Comment:
Not to self. I need to double check this was actually needed. Seems fishy. Not a blocker for merge to feature branch, but I should double check before this merges to main.

Reply:
I wasn't able to resolve this, so leaving it in there for now.